### PR TITLE
feat(user): Add followup-creating functionality.

### PR DESCRIPTION
**Pull request by arxanas**
Hi! It didn't look like the ability to post a follow-up existed, so I added it. Here's a picture of it in action:

screen shot 2015-05-23 at 8 52 14 pm

### CODE REVIEW

File: piazza_api/network.py [hunk 1]
```diff
@@ -105,6 +105,40 @@ def iter_all_posts(self, limit=None):
...
+    def create_followup(self, post, content, anonymous=False):
+        """Create a follow-up on a post `post`.
+
+        It seems like if the post has `<p>` tags, then it's treated as HTML,
+        but is treated as text otherwise. You'll want to provide `content`
+        accordingly.
+
+        :type  post: dict|str|int
+        :param post: Either the post dict returned by another API method, or
+            the cid` field of that post.
```
Comment (hfaran) [inline piazza_api/network.py]: Missing a "`" before `cid`

File: piazza_api/rpc.py [hunk 2]
```diff
@@ -96,6 +96,22 @@ def content_get(self, cid, nid=None):
...
+        """Create a post or followup.
+
+        :type  params: dict
+        :param params: A dict of options to pass to the endpoint. Depends on
+            the specific type of content being created.
+        :returns: Python object containing returned data
+        """
+        r = self.request(
+            method="content.create",
+            # blank.
```
Comment (hfaran) [inline piazza_api/rpc.py]: Not quite sure what this comment means?

**Comment by hfaran** [comment]
Just a couple comments but looks good otherwise! Thanks for adding this!

**Comment by arxanas** [comment]
Updated. (The `blank` comment was below another comment line, which I had deleted.)

I need this functionality for my own app, so I bumped the version number. Will it be retrievable from PyPI?

**Comment by hfaran** [comment]
I actually think we can make a `0.5.0` release because there are a few other commits that have not been merged into master and released yet. So if you can actually remove the version number change, I will merge this, and then bump the version number and release separately.

**Comment by arxanas** [comment]
Reverted. (Do you get notifications when I commit? I assume not.)

**Comment by hfaran** [comment]
I would get notifications, but since your are committing on your own fork that I am not watching, I do not. So it's fine if you just write a comment here telling me you updated like you did just now. :)

Okay so I will merge this in now and create a release, and upload it to PyPI.

**Comment by hfaran** [comment]
`0.5.0` is now available on PyPI. Also I added you to [CONTRIBUTORS.md](https://github.com/hfaran/piazza-api/blob/develop/CONTRIBUTORS.md); definitely feel free to add more in the future and drop your additions in here as well! :)

**Comment by arxanas** [comment]
Great, thanks :D

**Pull request merged**